### PR TITLE
Update switch case statement roles

### PR DESCRIPTION
**Pull request by BFergerson**
This makes the case statement roles more similar to Java's (https://github.com/bblfsh/java-driver/blob/00a490c36cd8516df428cbe40faed603b0b2b915/driver/normalizer/annotation.go#L536).

There is one difference now though and that's the additional `CASE` role with the `DEFAULT` role that's on PHP but not in Java. Should PHP be changed so the `CASE` is removed and it just has the `DEFAULT` or does the Java version need to be updated to add the missing `CASE` role?

---
This change is [Reviewable](https://reviewable.io/reviews/bblfsh/php-driver/56)

### CODE REVIEW

File: driver/normalizer/annotation.go [hunk 1]
```diff
@@ -446,9 +446,9 @@ var Annotations = []Mapping{
 	AnnotateType(php.Case, FieldRoles{
 		"cond":  {Opt: true, Roles: role.Roles{role.Case, role.Condition}},
 		"stmts": {Arr: true, Roles: role.Roles{role.Case, role.Body}},
-	}, role.Case),
+	}, role.Statement, role.Switch, role.Case),
```
Comment (dennwc) [inline driver/normalizer/annotation.go]: I'm not sure it's a `Statement` though. Other roles are correct, thanks!

**Comment by BFergerson** [comment]
@dennwc, there is still a difference between the Java version and the PHP version.

Please see:
> There is one difference now though and that's the additional `CASE` role with the `DEFAULT` role that's on PHP but not in Java. Should PHP be changed so the `CASE` is removed and it just has the `DEFAULT` or does the Java version need to be updated to add the missing `CASE` role?

**Comment by dennwc** [comment]
@BFergerson Yes, sure, let me review the annotations more carefully.

**Comment by dennwc** [comment]
@BFergerson https://github.com/bblfsh/sdk/pull/427

**Pull request merged**